R/msg.R

```diff
@@ -279,6 +288,19 @@ error_frame_matrix_list <- function(pos) {
   ))
 }
 
+error_tibble_row_size_one <- function(j, name, size) {
+  if (name != "") {
+    desc <- tick(name)
+  } else {
+    desc <- paste0("at position ", j)
+  }
+
+  tibble_error(bullets(
+    "All vectors in `tibble_row()` must be size one, use `list()` or `list_of()` to wrap.",
```

Comment:
Probably best just to recommend `list()` for now, `list_of()` is a bit too experimental still